types/networks/src/lib.rs

```diff
@@ -111,12 +42,56 @@ pub enum Height {
     Skyr,
 }
 
+pub const HEIGHT_VEC: [Height; 17] = [
+    Height::Breeze,
+    Height::Smoke,
+    Height::Ignition,
+    Height::ActorsV2,
+    Height::Tape,
+    Height::Liftoff,
+    Height::Kumquat,
+    Height::Calico,
+    Height::Persian,
+    Height::Orange,
+    Height::Trust,
+    Height::Norwegian,
+    Height::Turbo,
+    Height::Hyperdrive,
+    Height::Chocolate,
+    Height::OhSnap,
+    Height::Skyr,
+];
+
 impl Default for Height {
     fn default() -> Height {
         Self::Breeze
     }
 }
 
+impl From<Height> for NetworkVersion {
+    fn from(height: Height) -> NetworkVersion {
+        match height {
+            Height::Breeze => NetworkVersion::V0,
+            Height::Smoke => NetworkVersion::V1,
+            Height::Ignition => NetworkVersion::V2,
+            Height::ActorsV2 => NetworkVersion::V3,
+            Height::Tape => NetworkVersion::V4,
```

Comment:
Bump everything such that `Breeze` is `V1` and `Tape` is `V5`. See the deleted `UPGRADE_INFOS` for ground truth.